src/Context/QuizContext/Quiz.type.ts

```diff
@@ -0,0 +1,58 @@
+export type Option = {
+	_id: string;
+	text: string;
+	/**
+	 * For right answer make this true
+	 */
```

Comment:
If you are providing documentation, make sure it is provided uniformly across all the types to maintain code consistancy.

Reply:
Yes, I will remove it for now.